### ISSUE TITLE
Cannot search text in Help pane

### ISSUE THREAD

**Issue by drammock** [spyder-ide/spyder#3159]
## Searching in help pane doesn't work

1. Make sure some documentation is showing in the help pane
2. Click on the documentation so it's focused
3. Press `Ctrl+f`
4. Type something in the search box

**Expected output**
should scroll to and highlight the first instance of matching text in the documentation.

**What I see instead**
Upon pressing `Ctrl+f` I see this in the internal console:
```
Traceback (most recent call last):
  File "/home/drmccloy/Builds/spyder/spyderlib/widgets/findreplace.py", line 203, in show
    text = self.editor.get_selected_text()
AttributeError: 'FrameWebView' object has no attribute 'get_selected_text'
```

Upon typing anything in the search box I see this:
```
Traceback (most recent call last):
  File "/home/drmccloy/Builds/spyder/spyderlib/widgets/findreplace.py", line 295, in text_has_been_edited
    self.find(changed=True, forward=True, start_highlight_timer=True)
  File "/home/drmccloy/Builds/spyder/spyderlib/widgets/findreplace.py", line 322, in find
    found = self.editor.find_text(text, changed, forward, case=case,
AttributeError: 'FrameWebView' object has no attribute 'find_text'
```

## Versions and main components

* Spyder 3.0.0dev (Commit: eb840fe) 
* Python 2.7.6 64bits, Qt 4.8.6, PyQt4 (API v2) 4.10.4 on Linux

## Dependencies
jedi >=0.8.1    :  0.9.0 (OK)
matplotlib >=1.0:  1.5.1+1643.g0423430 (OK)
nbconvert >=4.0 :  4.2.0 (OK)
numpy >=1.7     :  1.12.0.dev0+2af06c8 (OK)
pandas >=0.13.1 :  0.18.0+104.g1c8816f (OK)
pep8 >=0.6      :  1.7.0 (OK)
psutil >=0.3    :  1.2.1 (OK)
pyflakes >=0.5.0:  1.1.0 (OK)
pygments >=1.6  :  2.1.3 (OK)
pylint >=0.25   :  1.5.5 (OK)
qtconsole >=4.0 :  4.2.1 (OK)
rope >=0.9.4    :  0.10.3 (OK)
sphinx >=0.6.6  :  1.5a0 (OK)
sympy >=0.7.3   :  None (NOK)

**Comment by drammock** [comment]
may or may not be related to #2458

**Comment by ccordoba12** [comment]
Thanks for reporting. We'll be sure to address this problem before we release Spyder 3.0 :-)

**Comment by ccordoba12** [comment]
I can reproduce this problem and I'll fix it in Spyder 3.0 beta4

**Comment by drammock** [comment]
fix tested / confirmed, thanks @ccordoba12!
